src/pages/about/useForm.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+function useForm(initialState) {
+  const [formdata, setFormdata] = React.useState(initialState)
+  const [errors, setErrors] = React.useState(initialState)
+
+  const handleChange = event => {
+    const value = event.target.value
+    const nextState = { ...formdata, [event.target.name]: value }
+    const nextErrorState = { ...errors, [event.target.name]: '' }
+    setFormdata(nextState)
+    setErrors(nextErrorState)
+  }
+
+  return {
+    formdata,
+    errors,
+    handleChange,
+    setErrors,
+    setFormdata
+  }
+}
+
+export default useForm
```

Comment:
This file should also be refactored to a `contactForm` folder in components. 2 space tabs here should be changes to 4

Reply:
I wonder if it's because of different IDE issues... Kat was the only one working on her own files so her file shouldn't have changed if it was just her. Hmmm I wonder what happened. Is it because of prettier?